components/script/dom/htmlinputelement.rs

```diff
@@ -1130,6 +1130,7 @@ impl TextControlElement for HTMLInputElement {
     }
 }
 
+#[allow(non_snake_case)]
```

Comment:
I don't think it's strictly necessary - I can see this in many `script/dom` files, so I've always followed those, also because I use an IDE which screams at me in yellow when I am working on a file where I don't add it:

![Screenshot 2024-11-14 at 5 07 49 pm](https://github.com/user-attachments/assets/c9284c28-9761-471c-8eb7-4e30c68ac537)

Happy to remove though, I should be able to override that somewhere in the IDE.

Reply:
Yeah, clippy complains when it's not present on the trait definition, but does not care about the impls. I'd prefer not to add them to the impls, if possible.

Reply:
Noted 👍 thank you.